lib/board_viewer.rb~

```diff
@@ -0,0 +1,18 @@
+class BoardViewer
+  def show(board)
+    value = board.regions.first.squares.first.number
```

Comment:
- Observe the Law of Demeter: "You can talk to your friends, but not your friend's friends!"